### ISSUE TITLE
webkit version in comparison to popular browsers

### ISSUE THREAD

**Issue by kmike** [annulen/webkit#568]
Hey,

Thanks again for the great work!

I have a rather stupid question: which webkit version is in Alpha 2 binaries (when using QtWebKit), in layman terms? Is it fair to say e.g. "similar to Safari from mid-2016"? Will it change in future releases an pre-releases, and how?

**Comment by annulen** [comment]
It corresponds to WebKitGTK 2.12.5 with backported patches on top of it. It's roughly equivalent to Safari 10, though Safari 10 has some additional web features as it was brached off a bit later, e.g. more ES6 and IndexedDB in Worker

After final 5.212.0 release WebKit will be upgraded to the current trunk

**Comment by kmike** [comment]
Got it, thanks!
